### ISSUE TITLE
Incompatibility with PHP 7.1

### ISSUE THREAD

**Issue by azngeek** [netz98/n98-magerun#881]
I get this error when running ```sys:setup:incremental````

```
Fatal error: Uncaught ArgumentCountError: Too few arguments to function Symfony\Component\Console\Helper\DialogHelper::askAndValidate(), 2 passed in phar:///usr/local/bin/magerun/src/N98/Magento/Command/System/Setup/IncrementalCommand.php on line 610 and at least 3 expected in phar:///usr/local/bin/magerun/vendor/symfony/console/Helper/DialogHelper.php on line 347

ArgumentCountError: Too few arguments to function Symfony\Component\Console\Helper\DialogHelper::askAndValidate(), 2 passed in phar:///usr/local/bin/magerun/src/N98/Magento/Command/System/Setup/IncrementalCommand.php on line 610 and at least 3 expected in phar:///usr/local/bin/magerun/vendor/symfony/console/Helper/DialogHelper.php on line 347

Call Stack:
    0.0013     353536   1. {main}() /usr/local/bin/magerun:0
    0.0323    2396160   2. N98\Magento\Application->run() /usr/local/bin/magerun:8
    0.3874    9791824   3. Symfony\Component\Console\Application->run() phar:///usr/local/bin/magerun/src/N98/Magento/Application.php:607
    0.3874    9791824   4. N98\Magento\Application->doRun() phar:///usr/local/bin/magerun/vendor/symfony/console/Application.php:126
    0.3883    9809360   5. Symfony\Component\Console\Application->doRun() phar:///usr/local/bin/magerun/src/N98/Magento/Application.php:575
    0.3884    9809360   6. Symfony\Component\Console\Application->doRunCommand() phar:///usr/local/bin/magerun/vendor/symfony/console/Application.php:195
    0.3886    9813672   7. N98\Magento\Command\AbstractMagentoCommand->run() phar:///usr/local/bin/magerun/vendor/symfony/console/Application.php:886
    0.3886    9813672   8. Symfony\Component\Console\Command\Command->run() phar:///usr/local/bin/magerun/src/N98/Magento/Command/AbstractMagentoCommand.php:441
    0.3888    9816472   9. N98\Magento\Command\System\Setup\IncrementalCommand->execute() phar:///usr/local/bin/magerun/vendor/symfony/console/Command/Command.php:259
    1.9606   15918360  10. N98\Magento\Command\System\Setup\IncrementalCommand->_listDetailedUpdateInformation() phar:///usr/local/bin/magerun/src/N98/Magento/Command/System/Setup/IncrementalCommand.php:85
    1.9607   15918360  11. Symfony\Component\Console\Helper\DialogHelper->askAndValidate() phar:///usr/local/bin/magerun/src/N98/Magento/Command/System/Setup/IncrementalCommand.php:610
```

As everything is from the phar-archive i guess that there are no other dependencies involved.

Tested on:

```
PHP 7.1.0 (cli) (built: Dec  6 2016 21:32:38) ( NTS )
Copyright (c) 1997-2016 The PHP Group
Zend Engine v3.1.0-dev, Copyright (c) 1998-2016 Zend Technologies
    with Xdebug v2.5.0, Copyright (c) 2002-2016, by Derick Rethans
```

**Comment by tkn98** [comment]
We believe that the bug reported is fixed in the development version. It can be upgraded to it using the self-update command with the --unstable switch.

**Comment by tkn98** [comment]
Thanks for the report, great to see that PHP 7.1 is much more strict here!

**Comment by ktomk** [comment]
Fix confirmed in #888 (dupe) by @bluec
